src/components/Footer.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <!-- Footer -->
+<footer class="page-footer font-small black-color" color="black">
+  <container class="text-left">
```

Comment:
I would investigate flexbot and use that instead of row and col. It's way easier and very intuitive and is the norm for placing elements